app/services/mailbluster_manager.rb

```diff
@@ -0,0 +1,40 @@
+class MailblusterManager
+  include HTTParty
+  base_uri 'https://api.mailbluster.com/api/leads/'
```

Comment:
If the `base_uri` has a trailing `/`, I think you can remove some of the string interpolation on in the later function calls.

Reply:
The trailing `/` is slightly misleading, because HTTParty is just parsing the URI to form a URI object. Hence, the string interpolation is necessary in later calls to create the correct URL.